app/controllers/passengers_controller.rb

```diff
@@ -0,0 +1,52 @@
+class PassengersController < ApplicationController
+  def index
+    trip = Trip.find_by(id: params[:trip_id])
+    @passengers = Passenger.all.order(:name)
+  end
+
+  def show
+    id = params[:id].to_i
+    @passenger = Passenger.find_by(id: id)
+    @trips = @passenger.trips.all
+    if @passenger.nil?
+      render :notfound, status: :not_found
+    end
+  end
+
+  def edit
+    @passenger = Passenger.find(params[:id].to_i)
+  end
+
+  def create
+    @passenger = Passenger.new(passenger_params)
+    if @passenger.save
+      redirect_to passengers_path
+    else
+      render :new
+    end
+  end
+
+  def new
+    @passenger = Passenger.new
+  end
+
+  def destroy
+    passenger = Passenger.find_by(id: params[:id].to_i)
+    # passenger.destroy
+    passenger.status = "inactive"
+    redirect_to passengers_path
+  end
+
+  def update
+    passenger = Passenger.find_by(id: params[:id].to_i)
```

Comment:
This isn't showing the user validation errors or asking them to fix the form.  That's because you don't have an `if` statement checking to see if the `update` method returned `true`.